src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -827,6 +827,9 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
     if location is None:
         location = rg_location
 
+    if client.get(resource_group_name, name):
+        raise CLIError('Cluster {} already exists in {}'.format(name, resource_group_name))
```

Comment:
```suggestion
        raise CLIError('Cluster {} already exists in {}'.format(name, resource_group_name))
```
Could write this as `raise CLIError(f'Cluster {name} already exists in {resource_group_name}')`
This is more readable and better way to substitute a String.